Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureNodejsWorkerRole/Worker.njsproj

```diff
@@ -1,46 +0,0 @@
-﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```

Comment:
So are you entirely removing this template?

Reply:
Yes, it's identical to `ProjectTemplates\TypeScriptAzureNodejsWorker\TypeScriptAzureNodejsWorker.vstemplate`